ocaml/typing/untypeast.ml

```diff
@@ -356,8 +356,9 @@ let pattern : type k . _ -> k T.general_pattern -> _ = fun sub pat ->
     | { pat_extra=[Tpat_type (_path, lid), _, _attrs]; _ } ->
         Ppat_type (map_loc sub lid)
     | { pat_extra= (Tpat_constraint ct, _, _attrs) :: rem; _ } ->
+        (* CR cgunn: recover mode constraint info here *)
```

Comment:
Thanks for putting down the CR. The good news is that we don't use this file, so it's fine to leave it out-of-sync.